src/DetailsScreen.tsx

```diff
@@ -0,0 +1,73 @@
+import { useQuery } from '@apollo/client';
+import React from 'react';
+import { ClipLoader } from 'react-spinners';
+import {DETAIL_QUERY} from './DetailsQuery';
+import {userId} from './UsersList';
+import {useHistory} from 'react-router-dom'; 
+
+export default function detailScreen (){
```

Comment:
Acho que fica mais claro explicitar que se trata de User né?

```suggestion
export default function UserDetailScreen() {
```